### ISSUE TITLE
roomsetting to strip HTML colours (as bridged from IRC)

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#5424]

**Comment by t3chguy** [comment]
+10000000

**Comment by t3chguy** [comment]
/me adds to 70 issue long todo list

**Comment by ara4n** [comment]
this is a particularly good candidate for granular settings, too - given you might want to enforce it for a room, or for an account, or even for a device (if devices have different themes).

**Comment by t3chguy** [comment]
Should this be just font colour attr or also `'data-mx-bg-color', 'data-mx-color'`

**Comment by genofire** [comment]
is there away to send different colors for light and dark themes?
e.g. for light themes, darkblue and for dark theme lightblue?

this is not readable on an element with dark theme enabled:
```html
<font color="blue">Info</font>
```

**Comment by t3chguy** [comment]
No, that is not something the Matrix spec allows @genofire - but it does allow setting a foreground and background colour so you can ensure legibility.

**Comment by genofire** [comment]
Hmm, should element render "default" colors dedicated by light and dark mode?
e.g. send-color: `<font color="{{ sendColor}}">Info</font>`

| send-color | light-theme | dark-theme | comment |
|------------|-------------|------------|---------|
| red        | red         | red        | always good readable |
| blue       | blue        | lightblue  ||
| green      | darkgreen   | green      ||
| yellow     | orange      | yellow     ||
